metis-authentication/metis-authentication-rest/src/main/java/eu/europeana/metis/authentication/rest/config/ApplicationConfiguration.java

```diff
@@ -122,7 +124,8 @@ public SessionFactory getSessionFactory(HibernateConfigurationProperties hiberna
     try (Session dbSession = sessionFactory.openSession()) {
       performAction(dbSession, session -> {
         Transaction tx = session.beginTransaction();
-        session.createNativeQuery(createTablesSql).executeUpdate();
+        NativeQueryImpl<Void> nativeQuery = new NativeQueryImpl<>(createTablesSql, (SharedSessionContractImplementor) session);
```

Comment:
I think that won't work with the current file that is executed.
From what I remember this method does not accept an sql "script" which contain multiple sql commands.
Have you tested it?

Reply:
I'm not able to test this, I'll just put it back to how it was before